src/common/utils.js

```diff
@@ -546,3 +547,20 @@ export const getCoresPerSocketPossibilities = (maxCoresPerSocket, vCPUs) => {
 
   return options
 }
+
+// Generates a random human-readable string of length `length`
+// Useful to generate random default names intended for the UI user
+export const getReadableRandomString = (() => {
+  const letterGetters = map([
+    [ 'a', 'e', 'i', 'o', 'u', 'y' ],
+    [ 'b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'z' ]
+  ], letters =>
+    () => letters[random(letters.length - 1)]
+  )
+
+  return function _ (length = 8) {
+    return length > 0
+      ? _(length - 1) + letterGetters[length % 2]()
+      : ''
+  }
+})()
```

Comment:
I find this implementation easier to understand:

```js
const getReadableRandomString = (() => {
  const CONSONANTS = 'bdfgklmnprtvz'.split('')
  const VOWELS = 'aeiou'.split('')
  return (length = 8) => {
    const result = new Array(length)
    for (let i = 0; i < length; ++i) {
      result[i] = sample((i & 1) === 0 ? VOWELS : CONSONANTS)
    }
    return result.join('')
  }
})()
```

Also, I removed unclear letters (letters that may sound the same in common languages).